hack/verify-spelling.sh

```diff
@@ -38,4 +38,4 @@ popd >/dev/null
 # All the skipping files are defined in hack/.spelling_failures
 skipping_file="${KUBE_ROOT}/hack/.spelling_failures"
 failing_packages=$(sed "s| | -e |g" "${skipping_file}")
-git ls-files | grep -v -e "${failing_packages}" | xargs misspell -i "Creater,creater,ect" -error -o stderr
+git ls-files | grep -v -e '\.go$' -e "${failing_packages}" | xargs misspell -i "Creater,creater,ect" -error -o stderr
```

Comment:
It would have been nice to get rid of verify-spelling.sh completely, but that doesn't work because golangci-lint only deals with Go files.